test.js

```diff
@@ -40,6 +40,14 @@ describe('markdown-it-implicit-figures', function() {
     assert.equal(res, expected);
   });
 
+  it('should add incremental tabindex to figures when opts.tabindex is set', function () {
```

Comment:
Let me know if you think the proposed test is sensible—I don't change the source or expected output, just render and compare twice.